server/core/workspace/workspace_update.go

```diff
@@ -50,6 +51,7 @@ func (h *UpdateWorkspaceHandler) Handle(ctx context.Context, query UpdateWorkspa
 
 	// data access
 	w := query.Workspace.DeepCopy()
+	log.FromContext(ctx).Info("updating workspace", "workspace", w)
```

Comment:
I suspect dumping the entire workspace is going to pollute our logs quite a bit.  Maybe we could prune it down a bit?

Reply:
good catch, I'd say we can use Debug here. WDYT?

Reply:
lgtm

Reply:
(done in cf201edff9b2211bd62fcb2fdeb4d17547619b89)